recursio-client/Characters/Player.gd

```diff
@@ -210,6 +219,7 @@ func handle_server_update(position, time):
 
 			Logger.info(("Corrected from " + str(before) + " to " + str(.get_position())
 				+ " (should be at " + str(position) + " according to server)"), "movement_validation")
+			print("WARNING: TODO check if this is still up to date (hotfix, lerp and correction) -> happens sometimes if switching timeline index very often")
```

Comment:
Is this up-to-date? If so, would be better as log and/or comment (with more detailed description)

Reply:
added an issue #161 and changed print to Logger.warn